packages/passport/sdk/src/overlay/constants.ts

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable max-len */
+
+export const PASSPORT_OVERLAY = 'passport-overlay';
+export const PASSPORT_OVERLAY_CLOSE = `${PASSPORT_OVERLAY}-close`;
+export const PASSPORT_OVERLAY_TRY_AGAIN = `${PASSPORT_OVERLAY}-try-again`;
+
```

Comment:
These classes don't seem to have any styles associated with them, and it looks like we're using them to retrieve particular elements.

Can I suggest that we use these as `id`'s instead and suffix these constants with `_ID` so that their purpose is clear?

Reply:
yep that makes sense 👍

Reply:
updated